runtime/src/lib.rs

```diff
@@ -122,6 +125,9 @@ pub type Executive = frame_executive::Executive<
     AllPalletsWithSystem,
 >;
 
+/// Type representing the storage data units in StorageHub.
+pub type StorageDataUnit = u32;
```

Comment:
This feels like something that should be defined in `configs/mod.rs` and imported here 🤔